src/styles/blocks/calendar.scss

```
@@ -0,0 +1,26 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  width: calc(
+    ($padding * 2) + ($day-size * $columns) + ($gap * ($columns - 1))
+  );
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: $font-size;
+    font-family: Arial, sans-serif;
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-bg;
+    border: $border-width solid $border-color;
+    transition: 500ms;
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+  }
+}
```

The month-length modifiers and hiding of days after the specified last day are missing. This violates checklist item 1.10: "Implement a month-length modifier with values 28, 29, 30, and 31 (31 by default in HTML) that hides days after the specified last day using nth-child selectors." It also violates checklist item 3.5: "The month-length modifier must hide or not render days beyond the specified length using nth-child-based selectors generated via @for." Add an @for loop to produce `.calendar--month-length-28`/29/30/31 rules that hide the excess day elements using `:nth-child(n)` selectors.